lighthouse-cli/test/smokehouse/perf/lantern-expectations.js

```diff
@@ -33,7 +33,8 @@ module.exports = [
       audits: {
         'interactive': {
           // Make sure all of the CPU time is reflected in the perf metrics as well.
-          score: '<.2',
+          // The scripts stalls for 3 seconds and lantern has a 4x multiplier so 12s minimum.
+          numericValue: '>12000',
```

Comment:
is the value just that these are more informative? Because from that screenshot in the other issue it seems like LR is going to be just as wrong with these numbers as the score numbers :)

Reply:
yes, just more informative.